### PR TITLE
Feat/some tunings

### PR DESCRIPTION
**Pull request by amimart**
Improve some existing flows:

## Publish

Now publish npm `next` versions to public registry.

The logic being identical with Github Package publication, the two jobs has been merged and refactored using matrix strategy.

## Lint

Mainly for PR from forks purposes:
- `report-new-dependencies`: Change Github token used to comment PRs.
- `lint-docker`: Remove `dockerlint` action, by annotating PR changes it needs a token with repository access which we can't ensure from forks. Being less strict than `hadolint` I think removing it is rational.

### CODE REVIEW

**Review by ccamel — APPROVED** [review]

File: src/.github/workflows/publish.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ concurrency:
 
 jobs:
   publish-docker-images:
-    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+    if: contains(github.ref, 'refs/tags/')
```
Comment (ad2ien) [inline src/.github/workflows/publish.yml:14]: Hmm, did this worked to generate nightly image in case of a push on main?

**Review by ad2ien — CHANGES_REQUESTED** [review]

File: src/.github/workflows/publish.yml (hunk 1 above)
Comment (ad2ien) [reply src/.github/workflows/publish.yml:14]: I just realized I removed this line in my pull request https://github.com/okp4/actions/pull/9

Comment (amimart) [reply src/.github/workflows/publish.yml:14]: Totally right ! I removed it as the event could only be a `push`, but the entire condition should be removed to allow nightly image publication.

**Review by hleonardtek — APPROVED** [review]

File: src/.github/workflows/publish.yml [hunk 2]
```diff
@@ -146,7 +121,7 @@ jobs:
             feat: update storybook site (auto)
 
   publish-pypi:
-    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+    if: contains(github.ref, 'refs/tags/')
```
Comment (ad2ien) [inline src/.github/workflows/publish.yml:124]: We don't want nightly builds for pyhton packages?

Comment (amimart) [reply src/.github/workflows/publish.yml:124]: Indeed, we currently have no strategy for publishing non-release python packages.

**Pull request merged**